app/dashboard/views.py

```diff
@@ -4044,3 +4047,103 @@ def choose_persona(request):
             'persona': persona,
         },
         status=200)
+
+
+@csrf_exempt
+@require_POST
+def join_tribe(request, handle):
+    print(handle, request.user.profile)
+    if request.user.is_authenticated:
+        profile = request.user.profile if hasattr(request.user, 'profile') else None
+        print(profile)
+        try:
+            TribeMember.objects.get(profile=profile, org__handle__iexact=handle).delete()
+            return JsonResponse(
+            {
+                'success': True,
+                'is_member': False,
+            },
+            status=200)
```

Comment:
^ do we need to pass success True ? cause status is 200

Reply:
yes more a rest pattern